citi-onboarding/celeiro-dos-bambas
# Feature/impacto-contato

## What I did

- Created Impacto using Strapi
- Created Contato using Strapi
   -
## How to test

1. Open the terminal and go to the folder of the project "celeiro-dos-bambas"

2. In terminal:
   2.1. `cd server`
   2.2. `code .`
   
3. In VSCode:

   3.1. `yarn develop`
   
4. In your browser go to <http://localhost>:1337/admin

5. Check if the creation looks just like it was supossed based on the Kanban on squad's Notion

6. Read the changed files section on GitHub

7. Test the things that you think are worth testing, even the ones that are not in this description
